classes/class-filter-input.php

```diff
@@ -35,8 +35,6 @@ class Filter_Input {
 		FILTER_SANITIZE_SPECIAL_CHARS => 'htmlspecialchars',
 		FILTER_SANITIZE_STRING        => 'sanitize_text_field',
 		FILTER_SANITIZE_URL           => 'esc_url_raw',
-		// Other.
-		FILTER_UNSAFE_RAW             => null,
```

Comment:
Both `FILTER_UNSAFE_RAW` and `FILTER_DEFAULT` are equal to `516` so that makes the last item override the value. However, they are both currently `null` so it doesn't actually change it.

Looks like this data structure would need to be reworked to account for potentially duplicate keys so I suggest we leave it as is until that is done.